<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>692</width>
    <height>386</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>671</width>
      <height>331</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SizeConstraint::SetMaximumSize</enum>
     </property>
     <item row="1" column="0">
      <widget class="QTableView" name="tableView"/>
     </item>
     <item row="2" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="pb_add_tick">
         <property name="text">
          <string>Взять билет</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pb_del_no_block">
         <property name="text">
          <string>Удал. неактив.</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pb_clear_route_dat">
         <property name="text">
          <string>Завершить</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pb_control">
         <property name="text">
          <string>Контроль</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pb_info">
         <property name="text">
          <string>Информация</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pb_del_tick">
         <property name="text">
          <string>Отменить билет</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="pb_save">
         <property name="text">
          <string>Сохранить</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pb_update">
         <property name="text">
          <string>Обновить</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pb_cancel">
         <property name="text">
          <string>Отмена</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pb_chek_data">
         <property name="text">
          <string>Переключить</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pb_to_file">
         <property name="text">
          <string>Отчёт</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>692</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Вход</string>
    </property>
    <addaction name="action_reg"/>
    <addaction name="action_enter"/>
    <addaction name="action_exit_u"/>
   </widget>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_reg">
   <property name="text">
    <string>Регистрация</string>
   </property>
  </action>
  <action name="action_enter">
   <property name="text">
    <string>Вход</string>
   </property>
  </action>
  <action name="action_exit_u">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
